### PR TITLE
2nd try , Flexbox page update 

### PR DESCRIPTION
**Pull request by IamManchanda**
Not a fan of duplication so here it is https://github.com/zurb/foundation-sites/pull/9598

But still will duplicate a bit :stuck_out_tongue_closed_eyes: 

@ncoden @kball .... just a ~~first~~ second ( **and safe** ) pull request .... 
if you happy with this change, approve and merge it, 
if you dont want this, simply close it :smiling_imp:

### CODE REVIEW

File: docs/pages/flexbox.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ sass:
   - scss/util/_flex.scss
 ---
 
-Foundation components use a combination of floats, vertical alignment, table cells, and various other CSS hacks to get layouts looking right. These days, there's a better way... if you have the browser support!
+Foundation components use a combination of floats, vertical alignment, table cells, and various other CSS hacks to get layouts looking right. These days though, there's a better way... if you are happy with the below browser support!!
```
Comment (kball) [inline docs/pages/flexbox.md]: Ok, one more piece of git/github practice for you. :)  You added an extra '!' at the end... lets remove it. You should be able to update this PR simply by pushing another commit on the same branch.

Comment (IamManchanda) [inline docs/pages/flexbox.md]: Just did that .... thanks a lot

**Comment by kball** [comment]
Ok awesome.  And one more lesson. There are 4 commits here, that are conceptually 1 commit... so I'd like them squashed down to 1 commit for the final PR. You can do this by using an interactive git rebase (http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html)

In this case, what you'll do is, from that branch, do

`git rebase -i master`

change all but the first commit to `s` or `squash`

And then do

`git push -f origin HEAD`

**Comment by IamManchanda** [comment]
Salute you kevin ball, you are simply awesome

**Pull request merged**